src/main/com/fulcrologic/fulcro/algorithms/do_not_use.cljc

```diff
@@ -150,3 +150,17 @@
     (if (vector? <>)
       <>
       [<>])))
+
+(defn- check-query-valid
+  "Runtime check that the (typically root) query looks valid
+  (though the Spec it uses might be both more strinct / more lax than the code
+  so do not take it absolutely.
+  "
+  [query component-name-fn]
+  (when (and #?(:clj true :cljs goog.DEBUG)
```

Comment:
should there be `:clj true` or false?

Reply:
doesn't really matter. Probably false, since you wouldn't run an SSR on a broken app, but whatever :D